### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by dennypc** [SyntaxC4-MSFT/JekyllExtension#5]
Hi, I'm trying to set it up on a Azure WebSite and keep getting this error:

Failed to install web app extension Jekyll. Heuristics indicate WebApiClient request timed out. Uri: https://dennypcsite.scm.azurewebsites.net/api/siteextensions/JekyllExtension
Timeout: 00:01:00

I've already specified SCM_COMMAND_IDLE_TIMEOUT=600 in App Settings and Restarted the App.

Thanks!

**Comment by fboucher** [comment]
I got the same error, but after 2 minutes or so, the extension was installed.
Did yours is?

**Comment by dennypc** [comment]
@FBoucher Hi, mine was not installed (well it says it was but it doesn't seem to be working).  Here's the error I get:

```
Command: D:\home\SiteExtensions\JekyllExtension\Hooks\deploy.cmd
'jekyll' is not recognized as an internal or external command,
operable program or batch file.
An error has occurred during web site deployment.
'jekyll' is not recognized as an internal or external command,\r\noperable program or batch file.\r\nD:\Program Files (x86)\SiteExtensions\Kudu\58.51007.2458\bin\Scripts\starter.cmd D:\home\SiteExtensions\JekyllExtension\Hooks\deploy.cmd
```

**Comment by dennypc** [comment]
Found out what the issue was, it's the all time favorite SSL CA issue: 

`Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed`

Here's what I did to work around it:
1. Download CA pem file to the root of my git repository (https://curl.haxx.se/ca/cacert.pem)
2. Connect Azure to my git repo (Under Deployment options)
3. Use Azure's debugconsole (https://<sitename>.scm.azurewebsites.net/DebugConsole)
4. Run the following command on the console: `set SSL_CERT_FILE=D:\home\site\repository\cacert.pem`
5. Navigate to the extension folder: D:\home\SiteExtensions\JekyllExtension\
6. Run install.cmd
7. Profit

Not sure what the fix would be for this extension? Maybe upgrading the RubyGems/DevKit version?
